pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -249,8 +249,11 @@ const (
 
 // Values for `VolumeType`
 const (
-	// NodeVolumeTypeGP2 is General Purpose SSD (default)
+	// NodeVolumeTypeGP2 is General Purpose SSD
 	NodeVolumeTypeGP2 = "gp2"
+	// NodeVolumeTypeGP3 is General Purpose SSD which can be optimised for
```

Comment:
ah looks like the json schema generator doesn't like newlines in these comments atm